.golangci.yml

```diff
@@ -36,7 +36,8 @@ linters-settings:
         alias: cerrdefs
       - pkg: github.com/opencontainers/image-spec/specs-go/v1
         alias: ocispec
-
+  goimports:
+    local-prefixes: github.com/docker
```

Comment:
We should probably make this `github.com/docker/docker` to have _actual_ local package only (also see my other comment)